File: Python/Level 1/약수의 개수와 덧셈 (구현).py
def measure(num):
    cnt = 0
    for i in range(1, num+1):
        if num % i == 0:
            cnt += 1

    return True if cnt % 2 == 0 else False


def solution(left, right):
    answer = 0
    for i in range(left, right+1):
        if measure(i):
            answer += i
        else:
            answer -= i

    return answer

File: Javascript/Level 1/약수의 개수와 덧셈 (구현).js
function check(num) {
	let cnt = 0;

	for (let i = 1; i <= num; i++) {
		if (num % i === 0) {
			cnt += 1;
		}
	}

	return cnt % 2 ? false : true;
}

function solution(left, right) {
	let answer = 0;

	for (let i = left; i <= right; i++) {
		answer += check(i) ? i : -i;
	}

	return answer;
}
